sdk/Sdk.Generators/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -0,0 +1,735 @@
+                    {
+                        bindingsList.Add(GetHttpReturnBinding(Constants.ReturnBindingName));
+                        hasOutputBinding = true;
+                    }
+                    else
+                    {
+                        var members = returnTypeSymbol.GetMembers();
+                        var foundHttpOutput = false;
+
+                        foreach (var m in members)
+                        {
+                            // Check if this attribute is an HttpResponseData type attribute
+                            if (m is IPropertySymbol property && SymbolEqualityComparer.Default.Equals(property.Type, Compilation.GetTypeByMetadataName(Constants.HttpResponseType)))
+                            {
+                                if (foundHttpOutput)
+                                {
+                                    _context.ReportDiagnostic(Diagnostic.Create(DiagnosticDescriptors.MultipleBindingsGroupedTogether, m.Locations.FirstOrDefault(), new object[] { nameof(m), nameof(returnTypeSymbol) }));
+                                    bindingsList = null;
+                                    return false;
+                                }
+
+                                foundHttpOutput = true;
+                                bindingsList.Add(GetHttpReturnBinding(m.Name));
+                            }
+                            else if (m.GetAttributes().Length > 0) // check if this property has any attributes
+                            {
+                                var foundPropertyOutputAttr = false;
+
+                                foreach (var attr in m.GetAttributes()) // now loop through and check if any of the attributes are Binding attributes
+                                {
+                                    if (SymbolEqualityComparer.Default.Equals(attr.AttributeClass?.BaseType?.BaseType, Compilation.GetTypeByMetadataName(Constants.BindingAttributeType)))
+                                    {
+                                        // validate that there's only one binding attribute per proeprty
+                                        if (foundPropertyOutputAttr)
+                                        {
+                                            _context.ReportDiagnostic(Diagnostic.Create(DiagnosticDescriptors.MultipleBindingsGroupedTogether, m.Locations.FirstOrDefault(), new object[] { nameof(m), nameof(returnTypeSymbol) }));
+                                            bindingsList = null;
+                                            return false;
+                                        }
+
+                                        IPropertySymbol? propertySymbol = m as IPropertySymbol;
+
+                                        if (propertySymbol is null)
+                                        {
+                                            throw new InvalidOperationException($"The property '{nameof(propertySymbol)}' is invalid.");
```

Comment:
Should we throw? Will that stop only our source generation? Will VS show any errors?